cmdutils/switches/switches.go

```diff
@@ -70,25 +106,25 @@ func (s *Switches) Set(val string) error {
 		// Validate that all specified controllers are known
 		for _, v := range settings {
 			trimmed := strings.TrimPrefix(v, disablePrefix)
-			if _, ok := s.settings[trimmed]; trimmed != All && !ok {
+			if _, ok := s.defaults[trimmed]; trimmed != All && !ok {
 				return fmt.Errorf("unknown item: %s", trimmed)
 			}
 		}
 	} else {
 		settings = []string{""}
 	}
 
-	s.setSettings(settings)
+	s.settings = s.prepareSettings(settings)
 	return nil
 }
 
 // Enabled checks if item is enabled
-func (s *Switches) Enabled(name string) bool {
+func (s Switches) Enabled(name string) bool {
```

Comment:
Why is the pointer receiver gone here & in `All`?

Reply:
So it can be used with `Switches` value and not only with pointer.

Reply:
If you have a pointer receiver, it also automatically works on bare structs by auto-dereferencing.